deposit_contract_proxy.sol

```diff
@@ -1,10 +1,18 @@
 pragma solidity ^0.6.4;
+pragma experimental ABIEncoderV2;
 
 interface DepositContract {
-    function deposit(bytes, bytes32, bytes, bytes32) external payable;
+    function deposit(
+        bytes calldata publicKey,
+        bytes32 withdrawalCredentials,
+        bytes calldata signature,
+        bytes32 depositDataRoot
+    ) external payable;
 }
 
 library DepositSSZ {
+    uint constant PUBLIC_KEY_LENGTH = 48;
```

Comment:
ahhh good catch -- ideally we don't duplicate this here but may be the pragmatic solution for now.

can we define constants at the file-level scope?

Reply:
Not yet, see https://github.com/ethereum/solidity/issues/3411.

Also file-level scoping was planned too (for structs and enums), but neither that is available.